src/sedtrails/transport_converter/plugins/physics/library/Fluid.py

```diff
@@ -0,0 +1,57 @@
+"""
+Class representing Fluid physical properties used by the transport converter.
+"""
+
+from dataclasses import dataclass
+import numpy as np
+
+
+@dataclass
+class Fluid:
+    """
+    For example, water and wind.
+    
+    Attributes
+    ----------
+    abstau_mean : float
+        absolute mean shear stress of the fluid [N/m^2].
+    abstau_max : float
+        absolute maximum shear stress of the fluid [N/m^2].
+    U_mag : float
+        total velocity magnitude of the fluid [m/s].
+    ustar_mean : float
+        mean friction velocity of the fluid [m/s].
+    ustar_max : float
+        maximum friction velocity of the fluid [m/s].
+    """
+    
+    # concrete method
+    def abstau_mean(self, D) -> float:

```

Comment:
Please, give a more descriptive name to D... and also add a docstring and type hint (if we know that already). Also do the same in the cases below